### PR TITLE
Profiling tool: Enable tests and update compressed event log

### PR DESCRIPTION
**Pull request by nartal1**
This enables the tests which were disabled after large files were removed from the repo.
This tests on a smaller compressed event log file. 
Closes https://github.com/NVIDIA/spark-rapids/issues/2644

Signed-off-by: Niranjan Artal <nartal@nvidia.com>

### CODE REVIEW

**Comment by nartal1** [comment]
build

File: tools/src/test/scala/com/nvidia/spark/rapids/tool/profiling/HealthCheckSuite.scala [hunk 1]
```diff
@@ -38,11 +38,10 @@ class HealthCheckSuite extends FunSuite {
   private val expRoot = ToolTestUtils.getTestResourceFile("ProfilingExpectations")
   private val logDir = ToolTestUtils.getTestResourcePath("spark-events-profiling")
 
-  /*
   test("test task-stage-job-failures") {
     var apps: ArrayBuffer[ApplicationInfo] = ArrayBuffer[ApplicationInfo]()
     val appArgs =
-      new ProfileArgs(Array(s"$logDir/task_job_failure_eventlog"))
+      new ProfileArgs(Array(s"$logDir/tasks_executors_fail_compressed_eventlog.zstd"))
```
Comment (tgravescs) [inline tools/src/test/scala/com/nvidia/spark/rapids/tool/profiling/HealthCheckSuite.scala:44]: nit, I think .zst is the normal extension to use for zstd.

Comment (tgravescs) [reply tools/src/test/scala/com/nvidia/spark/rapids/tool/profiling/HealthCheckSuite.scala:44]: nevermind, looks like spark uses .zstd

**Review by tgravescs — APPROVED** [review]

**Comment by tgravescs** [comment]
build

**Pull request merged**